src/sentry/static/sentry/app/routes.jsx

```diff
@@ -159,16 +159,6 @@ const orgSettingsRoutes = [
     component={errorHandler(OrganizationIntegrations)}
   />,
 
-  <Route key="members" path="members/" name="Members">
```

Comment:
This needs to remain in this array because it's used in multiple places. Any reason why it wouldn't work if it's declared here?

Reply:
I added a comment where I moved this. I need access to the hook store to determine which component to render, and the hook store isn't available at import-time.

Reply:
Ah, in that case can we just move the `orgSettingsRoutes` (+ project) declarations inside of `routes()`?

Reply:
Sure, would it be cleaner if I just added the member route in the function?

Reply:
Sure, the members routes just needs to be declared in `orgSettingsRoutes`.